examples/fluent-bit/cloudwatchlogs-emf/README.md

```diff
@@ -0,0 +1,119 @@
+    # including the HOSTNAME is a way to increase the number of LogStreams.
+    # The maximum throughput on a
+    # single LogStream is 5 MB/s (max 1 MB at max 5 TPS).
+    # In AWSVPC mode, the HOSTNAME is the ENI private IP
+    # in bridge mode, the HOSTNAME is the Docker container ID
+
+# Output for EMF over TCP -> CloudWatch
+[OUTPUT]
+    Name                cloudwatch
+    Match               emf-*
+    region              us-west-2
+    log_key             log
+    log_group_name      aws-emf-ecs-firelens-example-metrics
+    log_stream_prefix   from-fluent-bit-
+    auto_create_group   true
+    log_format          json/emf
+```
+
+Fluent Bit will accept EMF over TCP port `25888`. Build this config file into a custom Fluent Bit image as shown in the base [config-file-type](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/config-file-type-file) example. 
+
+Now we configure the EMF libraries in your app to send EMF to TCP port `25888`/ The setup steps are similar to using the [CW Agent to collect EMF in ECS](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Generation_CloudWatch_Agent.html). 
+
+#### Bridge Mode
+
+If your network mode is `bridge`, you can add the following env var to your app container definition to tell the library where to emit EMF. This env var uses a Docker hostname inserted by the link to the firelens container:
+
+```
+"name": "app",
+"links": [ "fluentbit" ], // the link is based on the container name
+"environment": [{
+                "name": "AWS_EMF_AGENT_ENDPOINT",
+                "value": "tcp://fluentbit:25888"
+              }]
+```
+
+And then setup a port mapping on the FireLens container:
+```
+"name": "fluentbit",
+"portMappings": [{
+                  "protocol": "tcp",
+                  "containerPort": 25888
+              }],
+```
+
+See the full task definition in the [emf-over-tcp](emf-over-tcp) directory.
+
+### AWSVPC or Host Mode
+
+If you are using host or AWSVPC network mode, the FireLens container can be reached on localhost. Note that for host mode, the host ENI is used, and so Fluent Bit must be the only process on the host using this port. 
+
+In this case, you simply set an env var on the app container for the EMF libraries:
+
+```
+"environment": [{
+                "name": "AWS_EMF_AGENT_ENDPOINT",
+                "value": "tcp://127.0.0.1:25888"
+              }],
+```
+
+## Tutorial 2: Send EMF to Fluent Bit via stdout
+
+If your app's EMF library is configured to send EMF json events to stdout, then you can simply use the FireLens log driver options to send it:
+
+```
+	"logConfiguration": {
+		"logDriver":"awsfirelens",
+		"options": {
+			"Name": "cloudwatch_logs",
+			"region": "us-west-2",
+			"log_key": "log",
+			"log_group_name": "emf-log-group/application",
+			"auto_create_group": "true",
+			"log_stream_name": "emf-",
+            "log_format": "json/emf",
```

Comment:
nit: might want to format this line properly

Reply:
Also I use log_stream_name here which is wrong, should be prefix